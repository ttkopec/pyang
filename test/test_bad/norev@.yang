module norev {
    namespace "urn:norev";
    prefix "norev";
    revision 2017-09-26 {
    }
}
