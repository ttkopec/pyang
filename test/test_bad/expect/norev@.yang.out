norev@.yang:1: warning: FILENAME_BAD_REVISION
